charts/mothership/templates/snapshot-workflow.yaml

```diff
@@ -0,0 +1,308 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: mothership-snapshot
+  namespace: {{ $.Release.Namespace }}
+spec:
+  schedule: "0 18 * * *"
+  concurrencyPolicy: "Forbid"
+  startingDeadlineSeconds: 7200
+  timezone: "Asia/Seoul"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 3 
+  workflowSpec:
+    entrypoint: mothership-snapshot
+    serviceAccountName: argo-workflows
+    volumeClaimTemplates:                 # define volume, same syntax as k8s Pod spec
+    - metadata:
+        name: snapshot-node-volume
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        storageClassName: {{ $.Values.storageClassName }}
+        resources:
+          requests:
+            storage: 100Mi
+    - metadata:
+        name: snapshot-data-volume
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        storageClassName: {{ $.Values.storageClassName }}
+        resources:
+          requests:
+            storage: 100Gi   
+
+    # This spec contains two templates: hello-hello-hello and whalesay
+    templates:
+    - name: mothership-snapshot
+      # Instead of just running a container
+      # This template has a sequence of steps
+      steps:
+      - - name: fetch-genesis-
+          template: fetch-genesis 
+      - - name: fetch-snapshot-
+          template: fetch-snapshot 
+      - - name: genesis-init-
+          template: genesis-init
+      - - name: op-stack-
+          template: op-stack
+      - - name: create-snapshot-
+          template: create-snapshot
```

Comment:
are `- -` and hypens after step names intended?

Reply:
https://argo-workflows.readthedocs.io/en/stable/workflow-concepts/#steps
i think we can use both `- -` and `-`. the former seems to make it execute multiple steps parallel.
and we dont need the last hyphen as it isnt used for prefix now